core/cosmosdb/cache-invalidator/src/main/scala/org/apache/openwhisk/core/database/cosmosdb/cache/Main.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.openwhisk.core.database.cosmosdb.cache
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import kamon.Kamon
+import org.apache.openwhisk.common.ConfigMXBean
+import org.apache.openwhisk.http.{BasicHttpService, BasicRasService}
+
+object Main {
+  def main(args: Array[String]): Unit = {
+    implicit val system: ActorSystem = ActorSystem("cache-invalidator-actor-system")
+    implicit val materializer: ActorMaterializer = ActorMaterializer()
+    ConfigMXBean.register()
+    Kamon.loadReportersFromConfig()
+    val invalidatorConfig = CacheInvalidatorConfig.getInvalidatorConfig()(system.settings.config)
+    val port = invalidatorConfig.port
+    //TODO HTTPS for ping/metric endpoint?
```

Comment:
how do you check liveness without ping?
the basic http service should be exposing /ping already no?

Reply:
Yes its uses basic http service so ping is supported. The purpose of `TODO` here was to determine if we need to expose the basic service over HTTPS or not. If yes then that would require copying the required cert files etc.